docs/configuration.md

```diff
@@ -411,6 +411,25 @@ Let's say that you are responsible for all accounts in the United States as well
 
 Now my account would represent a custom filter boolean dimension.
 
+##### Quantiles
+
+If you have dimension defined as histogram, you can add quantile measure. Use plywood method quantile on desired histogram and provide required parameters.
+Percentile parameter would be used as default percentile and could be adjusted on UI. Tunning parameters will be passed as is to druid. 
+
+```yaml
+- name: clicks_percentile
+  formula: $main.quantile($clicks, 0.99, 'k=128')
+```
+
+Turnilo can handle percentiles only as top level operation in expression so it is impossible to nest quantile expression inside let's say division. 
+
+```yaml
+- name: opaque_percentile_formula
+  forumla: $main.quantile($clicks, 0.9, 'k=128') * $user_weight
```

Comment:
hehe, clicks * user_weight :) I would change the examples to something more realistic. Eg:
$main.quantile($response_time_ms, ...)
$main.quantile($response_time_ms, ...) * 1000